scripts/diawi-upload.mjs

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env node
+
+import https from 'node:https'
+import { basename } from 'node:path'
+import { promisify } from 'node:util'
+import { createReadStream } from 'node:fs'
+
+import log from 'npmlog'
+import FormData from 'form-data'
+
+const UPLOAD_URL = 'https://upload.diawi.com/'
+const STATUS_URL = 'https://upload.diawi.com/status'
+const POLL_MAX_COUNT = 10
+const POLL_INTERVAL_MS = 700
+
+const sleep = (ms) => {
+  return new Promise((resolve, reject) => {
+    setTimeout(resolve, (ms))
+  })
+}
+
+const getRequest = async (url) => {
+  return new Promise((resolve, reject) => {
+    let data = []
+    https.get(url, res => {
+      res.on('error', err => reject(err))
+      res.on('data', chunk => { data.push(chunk) })
+      res.on('end', () => {
+        let payload = Buffer.concat(data).toString()
+        resolve({
+          code: res.statusCode,
+          message: res.statusMessage,
+          payload: payload,
+        })
+      })
+    })
+  })
+}
+
+const uploadIpa = async (ipaPath, comment, token) => {
+  var form = new FormData()
```

Comment:
Tiny nit pick over here :) 
It is better to use `const` or `let` for variable declarations that have block scope.

Reply:
Ah yeah, must have copy pasted that.